### PR TITLE
Fix to remove a dataset from a space #349

### PR DESCRIPTION
**Pull request by ddey2**
Signed-off-by: Dipannita Dey <ddey2@illinois.edu>

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [x] When possible

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the CHANGELOG.md.
- [ ] I have signed the CLA
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: app/services/mongodb/MongoDBSpaceService.scala [hunk 1]
```diff
@@ -424,12 +424,11 @@ class MongoDBSpaceService @Inject() (
    * @param space space id
    */
   def removeDataset(dataset:UUID, space:UUID): Unit = {
-    log.debug(s"Space Service - removing $dataset from $space")
+    log.info(s"Space Service - removing $dataset from $space")
```
Comment (robkooper) [inline app/services/mongodb/MongoDBSpaceService.scala:427]: can this be debug again?

**Comment by robkooper** [comment]
can you update the CHANGELOG, and add a line that due to this bug, counts might be off and users should run the fixcounts script

**Review by robkooper — CHANGES_REQUESTED** [review]
- changelog
- info -> debug

File: app/services/mongodb/MongoDBSpaceService.scala (hunk 1 above)
Comment (ddey2) [reply app/services/mongodb/MongoDBSpaceService.scala:427]: My apology. Fixed.

**Review by longshuicy — APPROVED** [review]

**Comment by ddey2** [comment]
> * changelog
> * info -> debug

Done

**Review by robkooper — APPROVED** [review]

**Pull request merged**